cp3pt0-deployment/setup_singleton.sh

```diff
@@ -231,7 +231,15 @@ function is_migrate_licensing() {
     fi
 
     title "Check migrating LTSR ibm-licensing-operator"
-    wait_for_licensing_instance_deployment
+    # wait for ibm-licensing-operator instance 
+    local licensing_operator_exist=$("$OC" get deployment -A | (grep ibm-licensing-operator || echo "fail"))  
+    if [[ $licensing_operator_exist != "fail" ]]; then
+        wait_for_licensing_instance_deployment
+    else
+        info "No ibm-licensing-operator found, skipping"
+        return 0
+    fi
+
     local ns=$("$OC" get deployments -A | grep ibm-licensing-service-instance | cut -d ' ' -f1)
```

Comment:
Sorry, one more comment, I noticed this line is also included in `wait_for_licensing_instance_deployment`, is it possible move this check under the above `if` condition? Thanks